Updated librairy (#3)
digitregroup/php-myelefant-client: README.md

You can send a campaign with the parameters provided when creating the campaign (message and sender) or by using custom parameters.

> **To create and send new campaign :** 

Reviewer: You can drop the `To` in order to use the imperative

- Sender :

   String
   
**To send sms with existing campaign**

> Important : To send sms with custom field, you must using tag when you creating campaign in MyElefant interface.

Reviewer 2: ```suggestion
Important : To send SMS messages with custom fields, you must use tags when you create the campaign in the MyElefant interface.
```

Example : 

[![Example](https://image.noelshack.com/fichiers/2019/08/4/1550760212-doc.png)](https://image.noelshack.com/fichiers/2019/08/4/1550760212-doc.png)

then, in the code :  

>         <?php

Reviewer 2: ```suggestion
```php
        <?php
```

        use myelefant\MyElefant;

        $client = new MyElefant(['secretKey' => '***SECRET_KEY***'])

        $client->sendSms(
                        'campaignId',

>                          ['33612345678','This content replace [[B]]','This content replace [[C]]','This content replace [[D]]']

Reviewer 2: ```suggestion
                         [
                            '33612345678',
                            'This content replace [[B]]',
                            'This content replace [[C]]',
                            'This content replace [[D]]'
                         ]
```

>                          ['33612345678','This content replace [[B]]','This content replace [[C]]','This content replace [[D]]']

Reviewer 2: If 33612345678 is a real campaign ID, please replace with a placeholder

Author: No, it's a phone number

                        );

> 

Reviewer 2: ```suggestion

\```

```
